File: Hurb/Hurb/Model/Suggestion.swift
//
//  Suggestion.swift
//  Hurb
//
//  Created by Alexandre Papanis on 10/08/19.
//  Copyright © 2019 Papanis. All rights reserved.
//

import Foundation

struct ResultSuggestion: Codable {
    let total: Int?
    let search: String?
    let suggestions: [Suggestion]?

}

struct Suggestion: Codable {
    let text: String?
    let suggestionType: String?
    let country: String?
    let state: String?
    let city: String?
    let sku: String?
    let filter: String?
    let url: String?
}

File: ChallengeAlpha/app/src/main/java/com/hurb/challengealpha/model/Suggestion.kt
package com.hurb.challengealpha.model

import com.google.gson.annotations.SerializedName

data class Suggestion(

    @SerializedName("text") val text: String,
    @SerializedName("suggestionType") val suggestionType: String,
    @SerializedName("country") val country: String,
    @SerializedName("state") val state: String,
    @SerializedName("city") val city: String,
    @SerializedName("filter") val filter: String,
    @SerializedName("url") val url: String
)